qa/acceptance/spec/shared_examples/cli/logstash-plugin/remove.rb

```diff
@@ -36,14 +36,13 @@
       end
     end
 
-    # Disabled because of this bug https://github.com/elastic/logstash/issues/5286
-    xcontext "when the plugin is installed" do
+    context "when the plugin is installed" do
       it "successfully removes it" do
         result = logstash.run_command_in_path("bin/logstash-plugin install logstash-filter-qatest")
         expect(logstash).to have_installed?("logstash-filter-qatest")
 
         result = logstash.run_command_in_path("bin/logstash-plugin remove logstash-filter-qatest")
-        expect(result.stdout).to match(/^Removing logstash-filter-qatest/)
+        # expect(result.stdout).to match(/^Removing logstash-filter-qatest/)
```

Comment:
Can we either remove the commented out line, or make this work? I'm fine with either - the text test is somewhat fragile, and the 'not_to_have_installed?` test is sufficient